server/src/test/java/com/lunchtime/controllers/RegisterPersonControllerTest.java

```diff
@@ -84,4 +90,18 @@ public void create_not_null_person_expected_true() throws Exception {
         PersonDto personDto = personService.saveRegisterPerson(registerPerson);
         Assert.assertNotNull(personDto);
     }
+
+    @Test
+    public void return_null_if_phone_number_equals_password() throws Exception {
+        registerPerson.setPassword(registerPerson.getPhoneNumber());
+        ResponseEntity<?> person = personController.createPerson(registerPerson);
+        Assert.assertNull(person);
+    }
+
+    @Test
+    public void return_null_if_email_equals_password() throws Exception {
```

Comment:
![image](https://user-images.githubusercontent.com/6152066/80211710-da7de700-863e-11ea-87da-0e1f77c60638.png)
![image](https://user-images.githubusercontent.com/6152066/80211857-20d34600-863f-11ea-912d-7d738b718c8d.png)

What this test do? It always green. Every developer can change creating person code, but your tests don't show error